_includes/v23.1/backups/backup-options.md

```diff
@@ -3,5 +3,5 @@
 `encryption_passphrase`<a name="with-encryption-passphrase"></a> | [`STRING`](string.html) | The passphrase used to [encrypt the files](take-and-restore-encrypted-backups.html) (`BACKUP` manifest and data files) that the `BACKUP` statement generates. This same passphrase is needed to decrypt the file when it is used to [restore](take-and-restore-encrypted-backups.html) and to list the contents of the backup when using [`SHOW BACKUP`](show-backup.html). There is no practical limit on the length of the passphrase.
 `detached`<a name="detached"></a>                           | [`BOOL`](bool.html) / None                   |  When a backup runs in `detached` mode, it will execute asynchronously. The job ID will be returned after the backup [job creation](backup-architecture.html#job-creation-phase) completes. Note that with `detached` specified, further job information and the job completion status will not be returned. For more on the differences between the returned job data, see the [example](backup.html#run-a-backup-asynchronously). To check on the job status, use the [`SHOW JOBS`](show-jobs.html) statement. Backups running on a [schedule](create-schedule-for-backup.html) have the `detached` option applied implicitly.<br><br>To run a backup within a [transaction](transactions.html), use the `detached` option.
-`kms`                                                            | [`STRING`](string.html) |  The [key management service (KMS) URI](take-and-restore-encrypted-backups.html#uri-formats) (or a [comma-separated list of URIs](take-and-restore-encrypted-backups.html#take-a-backup-with-multi-region-encryption)) used to encrypt the files (`BACKUP` manifest and data files) that the `BACKUP` statement generates. This same KMS URI is needed to decrypt the file when it is used to [restore](take-and-restore-encrypted-backups.html#restore-from-an-encrypted-backup) and to list the contents of the backup when using [`SHOW BACKUP`](show-backup.html). <br/><br/>Currently, AWS KMS and Google Cloud KMS are supported.
+`kms`                                                            | [`STRING`](string.html) |  The [key management service (KMS) URI](take-and-restore-encrypted-backups.html#uri-formats) (or a [comma-separated list of URIs](take-and-restore-encrypted-backups.html#take-a-backup-with-multi-region-encryption)) used to encrypt the files (`BACKUP` manifest and data files) that the `BACKUP` statement generates. This same KMS URI is needed to decrypt the file when it is used to [restore](take-and-restore-encrypted-backups.html#restore-from-an-encrypted-amazon-s3-backup) and to list the contents of the backup when using [`SHOW BACKUP`](show-backup.html). <br/><br/>Currently, AWS KMS and Google Cloud KMS are supported.
```

Comment:
> Currently, AWS KMS and Google Cloud KMS are supported.

We can arguably drop "currently", however, is Azure missing here?

Reply:
Oh yeah, good catch. Dropped "currently", added Azure